### ISSUE TITLE
OAuth authentication stops working.

### ISSUE THREAD

**Issue by gregkoehler** [osTicket/osTicket#6632]
Currently I have users that will get "Stuck" at the login screen when using Azure to login. This happens on both a new instance as well as mine. I have not seen any bug fixes in the next release up addressing this issue. This happens on the edge browser and if you delete the OSTicket OSTSESSID cookie it fixes the problem. The users will hit the authenicate with Azure button but the screen just comes back to the same login screen. This is running on an Ubuntu server.

![image](https://github.com/osTicket/osTicket/assets/5588556/88441baa-e6c9-4b75-aaa8-876f5f3369b8)

**Comment by JediKev** [comment]
@gregkoehler 

Yes, session issues is a known issue and is on our list to look into.

Cheers.

**Comment by mo-g** [comment]
Out of curiosity; was there any progress on this since? This is something that keeps hitting us; every time someone's source IP changes, they get trapped in a login-page loop until they either return to their original source IP or clear their OSTSESSID cookie.

We're on 1.18.3 FWIW.
